tests/Scripts/canonify-spicy-debug

```diff
@@ -5,4 +5,5 @@
 cat \
   | sed 's/stream=0x[0-9a-z]\{2,\} /stream=0xXXXXXXXX /g' \
   | sed 's/for stream 0x[0-9a-z]\{2,\}/for stream 0xXXXXXXXX/g' \
-  | sed 's/\(memory *: \).*/\1XXX/g'
+  | sed 's/\(memory *: \).*/\1XXX/g' \
+  | grep -v "^# user_time"
```

Comment:
This would omit any time information or the complete resource usage information if you follow my suggestion above.

Maybe we could insyead canonify time information similar to how we handle `memory` (canonify it, and not just drop it).